src/utils_project.js

```diff
@@ -34,8 +34,134 @@ go.utils_project = {
 
 // QUIZ HELPERS
 
-    has_untaken_quizzes: function() {
-        return false;  // returning false as temporary measure
+    // returns an array of untaken quizzes
+    get_untaken_quizzes: function(im) {
+        var endpoint = "quiz/untaken";
+        var params = {
+            "identity": im.user.answers.user_id
+        };
+        return go.utils
+            .service_api_call("quizzes", "get", params, null, endpoint, im)
+            .then(function(json_get_response) {
+                return json_get_response.data.results;
+        });
+    },
+
+    get_quiz: function(im, quiz_id) {
+        var endpoint = "quiz/"+quiz_id+"/";
+        return go.utils
+            .service_api_call("quizzes", "get", {}, null, endpoint, im)
+            .then(function(json_get_response) {
+                return json_get_response.data;
+        });
+    },
+
+    get_quiz_question: function(im) {
```

Comment:
for some or other reason passing in the question uuid does not work... it's as if the state gets loaded twice and by the second time the passed variable resets to 'undefined'?!  I've tried a couple of ways to pass it in...